packages/medusa/src/services/line-item.ts

```diff
@@ -332,6 +332,15 @@ class LineItemService extends TransactionBaseService {
       unit_price = context.variantPricing?.calculated_price ?? undefined
     }
 
+    if (unit_price == null) {
+      throw new MedusaError(
+        MedusaError.Types.INVALID_DATA,
+        `unit_price is required for variant item: ${
```

Comment:
@olivermrbl I am not sure about the wording, wdyt?

Reply:
Think this is good. Minor suggestion: `Cannot generate line item for variant ${} without a price.`